packages/eslint-plugin-svelte/tests/fixtures/rules/no-add-event-listener/invalid/test01-input.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+	const handler = (ev) => {
+		console.log(ev);
+	};
+
+	function onClick(event) {
+		const target = event.currentTarget;
+
+		const deepObj = {
+			deep: {
+				obj: {
+					target
+				}
+			}
+		};
+
+		target.addEventListener('focus', handler);
+		deepObj.deep.obj.target.addEventListener('focus', handler);
+	}
+
+	addEventListener('message', handler);
+	window.addEventListener('message', handler);
+	document.addEventListener('visibilitychange', handler);
```

Comment:
Do we need to add tests for 3rd argument?
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener